beastling/clocks/relaxed.py

```diff
@@ -2,22 +2,12 @@
 from beastling.util import xml
 
 
-def relaxed_clock_factory(clock_config, global_config):
-    distribution = clock_config.get("distribution","lognormal").lower()
-    if distribution == "lognormal":
-        return LogNormalRelaxedClock(clock_config, global_config)
-    elif distribution == "exponential":
-        return ExponentialRelaxedClock(clock_config, global_config)
-    elif distribution == "gamma":
-        return GammaRelaxedClock(clock_config, global_config)
-
-
 class RelaxedClock(BaseClock):
+    __type__ = 'relaxed'
 
     def __init__(self, clock_config, global_config):
-
         BaseClock.__init__(self, clock_config, global_config)
-        self.number_of_rates = int(clock_config.get("rates","-1"))
+        self.number_of_rates = int(clock_config.options.get("rates","-1"))
```

Comment:
Whether config options for clocks are explicitly listed in `sections.Clock` or simply pulled into the "catchall" `dict` `sections.Clock.options` seems a bit arbitrary now. The only guiding principle is that whatever needs a non-default `getter` becomes a proper attribute of the class. I think that's ok for now, but once we move to a proper plugin architecture, where other packages can contribute models and clocks, this might need to be more principled.